[2025-10-24T09:52:29.597+0000] {scheduler_job_runner.py:781} ERROR - Executor reports task instance <TaskInstance: load_csv_to_postgres_dag.spark_load_csv_to_postgres manual__2025-10-24T09:52:28.438165+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
